build.gradle

```diff
@@ -0,0 +1,100 @@
+import java.util.regex.Pattern
+
+plugins {
+    id 'java-library'
+}
+
+ant.importBuild('build.xml') { String oldTargetName ->
+    switch (oldTargetName) {
+        case 'javadoc': return 'ant_javadoc'
+        case 'init': return 'ant_init'
+        case 'jar': return 'ant_jar'
+        case 'dist': return 'ant_dist'
+        case 'increaseBuildNumber': return 'ant_increaseBuildNumber'
+        default: return oldTargetName
+    }
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = [ 'src' ]
+            outputDir = file('classes')
+        }
+    }
+    test {
+        java {
+            srcDirs = [ 'test' ]
+        }
+    }
+}
+
+test {
+    useJUnitPlatform()
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.apache.ant:ant:1.10.6'
+    implementation 'org.apache.axis2:axis2-adb:1.7.9'
```

Comment:
This replaces the xalan2 with a newer alternative. 👍 
I'd like to run some use cases to make sure it produces the same output.

Reply:
Re-introduced xalan now, my bad. :-)